src/main/java/com/google/sps/servlets/NaturalLanguageServlet.java

```diff
@@ -45,20 +48,30 @@
     @Override
     public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
 
+        boolean isMocking = false;
```

Comment:
Ideally the test should not impact the business logic. Please only mock the response in tests.

Reply:
Removed mentions of mocking, and now uses a boolean from a HttpServletRequest to determine if metadata should be included in the output.